### ISSUE TITLE
19.4.3d is an older version and will be depreciated soon

### ISSUE THREAD

**Issue by TimLavelle** [oracle/content-and-experience-toolkit#6]
Good Morning! I just installed the cec toolkit on my home dev server and when using the CLI, I get the following notification:

```
*******************************************************
* A newer version of Sites Toolkit CLI is available.  *
* You are using 19.4.3d. The latest is 20.1.1.        *
* Your current version will be deprecated soon.       *
*******************************************************
```

Looking at the commit logs, 19.4.3d was recently added so just want to know if this notification is a mistake or if there is a new version we can update to?

**Comment by jstortz** [comment]
Tim,

The ToolKit looks at the current version of OCE (which was updated to 20.1.1 this last week). It compares that to the version defined in this project and when it is behind it spits out this message once. When get get this repo updated to 20.1.1 or newer people will not encounter that message. I would go so far as to say, per how it is coded, it is expected behavior to see this. I also saw it. It also happened when 19.4.3 was released.

It is intended to get people to update their installed version of the toolkit, but it isn't a bullet-proof approach in situations where there is a little gap between when the cloud is updated and when the toolkit is updated.

Hopefully that makes some sense.

As of this writing, you are on the latest version and all set to go.
